js/dimensions.js

```diff
@@ -136,8 +136,12 @@ $(document).ready(function() {
 
   $(document).on('change', '#dimension_image_upload', function() { Resize.readFile(this); });
   $(document).on('change', '#resize-select', function() { Resize.fillDimensionFields(this); });
-  $(document).on('click', '.preview-result', function() { Resize.showResult(); });
   $(document).on('click', '.submit-btn', function(event) { Resize.downloadableResult(); });
+  $(document).on('click', '.preview-result', '#user-width, #user-height', function() {
+    if (($("#user-height").val() <= 400 && $("#user-width").val() <= 400) && (Resize.originalHeight > 800 && Resize.originalWidth > 800)) {
+      alert("The image is too big to crop to the desired dimension. Please resize your image to 600x600 or under via photoshop or gimp.");
```

Comment:
`600x600` value seems inconsistent with line above. Also, these values should probably be set as constants near top of file.

I would format line as follows:

    This image is too large to resize without distortions.\n\nYou will need to resize the image to be no larger than <W>x<H> using a photo editor like Photoshop or GIMP.

It is also possible to do the resize but just display a warning before showing the final result. I actually think that's a better approach. But check with product owner.

Reply:
Ah, that's a typo. My bad. I'll make the constants and edit the alert.

I'll double check with Ross on displaying the image with a warning or not displaying it at all.